Implement protocol test and boost bootstrapping by using cache (#665)
planetarium/libplanet: CHANGES.md

    the `Block<T>` more than once.  [[#647]]
 -  `Swarm<T>.PreloadAsync()` became to try downloading blocks from all neighbor peers,
    even if any peer among them is unavailable to send blocks.  [[#636]]

>  -  `Swarm<T>` became to compare only peer's address instead of public key

Reviewer: ```suggestion
  -  `Swarm<T>` became to compare only peer's addresses instead of public keys
```

>     when determining whether a peer is in routing table or not.  [[#664]]

Reviewer 2: ```suggestion
    when determining whether a peer is in routing table or not.  [[#665]]
```

>     when determining whether a peer is in routing table or not.  [[#664]]

Reviewer: ```suggestion
     to determine if a peer is in routing table or not.  [[#664]]
```

[#647]: https://github.com/planetarium/libplanet/pull/647
[#654]: https://github.com/planetarium/libplanet/pull/654
[#655]: https://github.com/planetarium/libplanet/pull/655

> [#664]: https://github.com/planetarium/libplanet/pull/664

Reviewer 2: ```suggestion
[#665]: https://github.com/planetarium/libplanet/pull/665
```
